### ISSUE TITLE
How do we _not_ transpile away class properties?

### ISSUE THREAD

**Issue by NullVoxPopuli** [embroider-build/embroider#1016]
This is related to: https://github.com/babel/ember-cli-babel/issues/417

Related embroider issues:
 - https://github.com/embroider-build/embroider/issues/979
 

In a default embroider build, class properties (non-decorated), should not be transpiled if they are targeting "last 1 firefox versions", but instead output properties like this:

```
 (0,_home_nullvoxpopuli_Development_NullVoxPopuli_limber_node_modules_embroider_core_node_modules_babel_runt
ime_helpers_esm_defineProperty__WEBPACK_IMPORTED_MODULE_0__.default)(
  this, 
  "rootURL", 
  _config_environment__WEBPACK_IMPORTED_MODULE_2__.default.rootURL
);
```

(this is the dev-time output, so its a bit verbose)

This is what I'd expect the output to be:
```
this.rootURL = _config_environment__WEBPACK_IMPORTED_MODULE_2__.default.rootURL
```

**Comment by NullVoxPopuli** [comment]
ah, this is controlled by ember-cli-babel! so once https://github.com/babel/ember-cli-babel/issues/417 is resolved, this, #1016, will be resolved

**Comment by NullVoxPopuli** [comment]
the answer: https://github.com/ef4/decorator-transforms
